.env.tmp

```diff
@@ -31,6 +31,7 @@ CONTAINER_REGISTRY_PASSWORD=""
 #
 
 EDGE_RUNTIME_VERSION="1.0"
+LAYERED_RUNTIME_TAG="1.0"
```

Comment:
>LAYERED_RUNTIME_TAG [](http://example.com/codeflow?start=0&length=19)

Should this be a boolean flag instead of integer version? Also, I would assume the default value should be false?

Reply:
My plan was to use this for the layered deployment versions, since they need to be used versions. On that note, I should rename this though, maybe LAYERED_DEPLOYMENT_VERSION?
We could add ENABLE_LAYERED deployment bool in addition, but I wasn't sure since it feels a bit clunky.